pkg/volume/util/quota/common/quota_linux_common.go

```diff
@@ -0,0 +1,77 @@
+// +build linux
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+// QuotaID -- generic quota identifier
+type QuotaID int32
```

Comment:
int32 is the underlying data type for a quota ID.  But I suppose that doesn't always have to be true.

Reply:
I see.  It is because the quota id is an int32 in linux: http://man7.org/linux/man-pages/man2/quotactl.2.html.  Might be worth linking to something in the comment.

Reply:
Done.